lib/screens/admin_order_detail_page.dart

```diff
@@ -0,0 +1,585 @@
+                  "注文者情報",
+                  style: TextStyle(
+                    fontSize: 18, color: paletteBlackColor, fontWeight: FontWeight.bold,
+                  )
+                ),
+              ),
+              SizedBox(height: 24.0),
+              _buildBuyerName(buyer),
+              SizedBox(height: 12.0),
+              _buildBuyerAddress(buyer),
+              SizedBox(height: 6.0),
+              _buildBuyerPhone(buyer),
+              SizedBox(height: 6.0),
+              _buildBuyerEmail(buyer),
+            ],
+          );
+        }
+      )
+    );
+  }
+
+  Widget _buildBuyerName(Customer buyer) {
+    return Container(
+      alignment: Alignment.centerLeft,
+      child: Row(
+        children: <Widget>[
+          Text(
+            buyer.name,
+            style: TextStyle(
+              fontSize: 16,
+              color: paletteBlackColor,
+              fontWeight: FontWeight.w600,
+            )
+          ),
+          Text(
+            " 様",
+            style: TextStyle(
+              fontSize: 16,
+              color: paletteBlackColor,
+              fontWeight: FontWeight.w300,
+            )
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildBuyerAddress(Customer buyer) {
+    return Container(
+      child: Row(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: <Widget>[
+          Text(
+            "住所",
+            style: TextStyle(
+              fontSize: 16, color: paletteBlackColor, fontWeight: FontWeight.w300,
+            ),
+          ),
+          SizedBox(width: 30.0),
+          Expanded(
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: <Widget>[
+                Text(
+                  // buyer.shippingAddresses[0].postCode
+                  "〒1580083",
+                  style: TextStyle(
+                    fontSize: 16, color: paletteBlackColor, fontWeight: FontWeight.w300,
+                  ),
+                ),
+                Text(
+                  // buyer.shippingAddresses[0].line
+                  "東京都世田谷区奥沢2-46-13",
+                  style: TextStyle(
+                    fontSize: 16, color: paletteBlackColor, fontWeight: FontWeight.w300,
+                  ),
+                ),
+              ],
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildBuyerPhone(Customer buyer) {
+    return Container(
+      alignment: Alignment.centerLeft,
+      padding: EdgeInsets.only( top: 6.0 ),
+      decoration: BoxDecoration(
+        border: Border(
+          top: BorderSide( color: Color(0xFFEFEFEF) ),
+        ),
+      ),
+      child: Row(
+        children: <Widget>[
+          Text(
+            "連絡先",
+            style: TextStyle(
+              fontSize: 16, color: paletteBlackColor, fontWeight: FontWeight.w300,
+            )
+          ),
+          SizedBox(width: 16.0),
+          Text(
+            // buyer.phoneNumber
```

Comment:
Same comment as before. I think we need to store this data obtained at the time of the order placement as part of the order model.